libnavigation-ui/src/test/java/com/mapbox/navigation/ui/internal/route/MapboxRouteLayerProviderFactoryTest.kt

```diff
@@ -1,5 +1,6 @@
 package com.mapbox.navigation.ui.internal.route
 
+import com.mapbox.navigation.ui.route.MapboxRouteLayerProviderFactory
```

Comment:
Can `MapboxRouteLayerProviderFactoryTest` test be move out of the `internal` package too?